CONTRIBUTING.md

```diff
@@ -127,35 +271,37 @@ The development cycle for qiskit-aqua is informed by release plans in the
  
 ### Branches
 
-* `master`
+* `master`:
 
-The master branch is used for development of the next version of qiskit-aqua.
+The master branch is used for development of the next version of qiskit-ignis.
 It will be updated frequently and should not be considered stable. The API
 can and will change on master as we introduce and refine new features.
 
-* `stable`
-
-The stable branch is used to maintain released versions of qiskit-aqua.
-It is tagged for each version of the code corresponding to the release of
-that version on PyPI. The only changes that will be merged to stable
-are for bug fixes. For further information please refer to the Qiskit
-[stable branch
- policy](https://qiskit.org/documentation/contributing_to_qiskit.html#stable-branch-policy)
+* `stable/*`:
+The stable branches are used to maintain the most recent released versions of
+qiskit-ignis. It contains the versions of the code corresponding to the minor
```

Comment:
Ignis -> Aqua

Reply:
Fixed in: https://github.com/Qiskit/qiskit-aqua/pull/1002/commits/656d12c0b86f93fc0490b749f56c52bce841068d